pkg/pillar/attest/attest.go

```diff
@@ -117,8 +117,9 @@ var (
 	ErrNonceMismatch       = errors.New("Nonce Mismatch")
 	ErrQuoteMismatch       = errors.New("Quote Mismatch")
 	ErrNoCertYet           = errors.New("No Cert Found")
-	ErrInfoTokenInvalid    = errors.New("Provided Integrity Token is Invalid")
+	ErrITokenMismatch      = errors.New("Provided Integrity Token is Invalid")
```

Comment:
Want to change the string to say”mismatch” as well?

Reply:
Done